collections.yml

```diff
@@ -201,30 +203,67 @@ Home Alone: &home_alone
 Bad Santa: &bad_santa
   - Bad Santa( 2)?
 Christmas:
-  - ^Die Hard$
   - *bad_santa
   - *home_alone
-  - White Christmas
-  - Christmas Vacation
-  - A Very Murray Christmas
-  - Office Christmas Party
-  - The Santa Clause
-  - Love Actually
-  - Krampus
-  - The Christmas Chronicles
+  - A Bad Moms Christmas
   - A Charlie Brown Christmas
-  - The Nightmare Before Christmas
+  - A Christmas Carol
+  - A Christmas Story
+  - A Very Harold (and|&) Kumar Christmas
+  - A Very Murray Christmas
   - Batman Returns
-  - The Family Stone
-  - The Ref
+  - Black Christmas
+  - Christmas Vacation
+  - Claymation Christmas
+  - Elf
+  - Four Christmases
+  - Frozen
```

Comment:
Please change `Frozen` to `Frozen(?! Planet)`; otherwise, it picks up the `Frozen Planet: xxxxxxxx` series.